angular/core/components/thread_page/proposal_outcome_panel/proposal_outcome_panel.coffee

```diff
@@ -3,13 +3,13 @@ angular.module('loomioApp').directive 'proposalOutcomePanel', ->
   restrict: 'E'
   templateUrl: 'generated/components/thread_page/proposal_outcome_panel/proposal_outcome_panel.html'
   replace: true
-  controller: ($scope, AbilityService, ProposalFormService) ->
+  controller: ($scope, AbilityService, ModalService, ProposalOutcomeForm) ->
 
     $scope.canCreateOutcome = ->
       AbilityService.canCreateOutcomeFor($scope.proposal)
 
     $scope.openProposalOutcomeForm = ->
-      ProposalFormService.openSetOutcomeModal($scope.proposal)
+      ModalService.open ProposalOutcomeForm, proposal: => $scope.proposal
```

Comment:
As an FYI, you don't need the `=>` here, because `$scope` is accessible here, whereas `@ / this` is not.

```
proposal: => @proposal # works
proposal: -> $scope.proposal # works
proposal: => $scope.proposal # works
proposal: -> @proposal # doesn't work, because '@ / this' is different when the method is called
```